styles.css

```diff
@@ -106,10 +115,410 @@ svg {
+}
+
+.recent_works_image {
+  width: 100%;
+}
+
+.recent_works_image img {
+  width: inherit;
+}
+
+/* styles for section1 works */
+
+.btw {
+  display: flex;
+  justify-content: center;
+}
+
+.ul_li {
+  font-size: 0.94em;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  line-height: 1.25em;
+  background-color: white;
+}
+
+.section1 {
+  width: 90%;
+  height: 100%;
+}
+
+.section1 p {
+  font-size: 1em;
+  line-height: 1.5em;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+}
+
+.section1 h1 {
+  color: #091e42;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 2em;
+  line-height: 2.75em;
+}
+
+.section1_button {
+  background-color: var(--cardbutton);
+  width: 7.75em;
+  height: 3em;
+  border: none;
+  margin-top: 0.375em;
+  padding: 0.75em;
+}
+
+.section1_button:hover {
+  background-color: #efa628fc;
+  border: 2px solid orangered;
+  cursor: pointer;
+
+  /* color: orange; */
+}
+
+.section1_button:active {
+  background-color: var(--cardbutton);
+}
+
+.section1_button:disabled {
+  background-color: #8993a4;
+}
+
+.section1 header h1 {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 2.5em;
+  line-height: 3.25em;
+  margin-top: 1.5em;
+}
+
+.section1 .horizontal_rule,
+.about .horizontal_rule {
+  width: 100%;
+  color: #212121;
+  margin: 0.625em 0 3.75em 0;
+}
+
+.section1_button a {
+  text-decoration: none;
+  color: white;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+}
+
+.section1 ul,
+.background_wrapper ul {
+  display: flex;
+  list-style: none;
+  margin-top: 0.375em;
+  margin-left: -0.25em;
+}
+
+.section1 ul li,
+.background_wrapper ul li {
+  padding: 4px;
+}
+
+.section1 ul li button,
+.background_wrapper ul li button {
+  padding: 0.63em 0.75em;
+  border: 1px solid #8993a4;
+}
+
+/* styles section 2 to 7 */
+
+.background {
+  display: flex;
+  justify-content: center;
+}
+
+.background_wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  width: 90%;
+  height: 100%;
+  background-image: url(img/background.png);
+}
+
+.background_wrapper .ul_li {
+  color: #fff;
+  background-color: rgba(255, 255, 255, 0.24);
+  margin-top: 20px;
+}
+
+.background_wrapper main,
+.background_wrapper section {
+  margin-left: 12px;
+}
+
+.background_wrapper h1 {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+  color: #fff;
+  line-height: 44px;
+}
+
+.background_wrapper p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 24px;
+  color: #fff;
+}
+
+.project {
+  width: 100%;
+  height: 3em;
+  background-color: var(--cardbutton);
+  border: none;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.project:hover {
+  background-color: #efa628fc;
+  border: 2px solid orangered;
+  cursor: pointer;
+}
+
+.project:active {
+  background-color: var(--cardbutton);
+}
```

Comment:
- [ ] Please make sure the background color and colors o f the hover and active state matches the design template.
### Your design
[Screencast from 25-03-2023 11:00:42.webm](https://user-images.githubusercontent.com/94551484/227710787-6cc1f5b5-843f-4742-9d37-df0d3a485e60.webm)
### Figma
![image](https://user-images.githubusercontent.com/94551484/227710801-d78c074c-f330-4133-82c6-ca52e8146bc9.png)